### DISCUSSION TITLE
Welcome to QuickShop-Reremake Discussions!

### DISCUSSION THREAD

**Discussion by Ghost-chu** [Ghost-chu/QuickShop-Reremake#755] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EfexTo** [comment]
Good evening, **after reporting 2 times** the issue about the big decimal numbers like 0.0001 i'm here again because ;

Even after enabling **disable-vault-format, decimal-format and setting the alternate-currency-symbol** when I try to buy from my-self an ITEM that cost 0.0003$, when I buy it gives me no money and say that I gained 3.0E-4...

**Please don't close the discussion,** I would like to use big decimal numbers and I think that I'm probably not the only one

**Reply by sandtechnology** [reply]
If you are trying buying yourself shop, it's normal for no money grows, since is the same source and destination, we can't bring memory from nothing, and the message is right, you DOES earned money from youself, expect no money changes.

**Reply by sandtechnology** [reply]
For the saying "3.0E-4", we will fix this issue in the next version

**Reply by EfexTo** [reply]
Ah, alright thanks
